src/test/java/io/strimzi/kafka/metrics/PrometheusMetricsReporterConfigTest.java

```diff
@@ -86,4 +86,21 @@ public void testValidator() {
         assertThrows(ConfigException.class, () -> validator.ensureValid(PrometheusMetricsReporterConfig.LISTENER_CONFIG, "randomhttp://:8080random"));
         assertThrows(ConfigException.class, () -> validator.ensureValid(PrometheusMetricsReporterConfig.LISTENER_CONFIG, "randomhttp://:8080"));
     }
+
+    @Test
+    public void testIsListenerEnabled() {
+        Map<String, Boolean> props = new HashMap<>();
+        props.put(PrometheusMetricsReporterConfig.LISTENER_ENABLE_CONFIG, true);
+        PrometheusMetricsReporterConfig config = new PrometheusMetricsReporterConfig(props);
+        assertTrue(config.isListenerEnabled());
+    }
+
+    @Test
+    public void testIsListenerDisabled() {
+        Map<String, Boolean> props = new HashMap<>();
+        props.put(PrometheusMetricsReporterConfig.LISTENER_ENABLE_CONFIG, false);
+        PrometheusMetricsReporterConfig config = new PrometheusMetricsReporterConfig(props);
+        assertFalse(config.isListenerEnabled());
```

Comment:
Can we also check that `startHttpServer()` return `Optional.empty()`?

Reply:
Added to the above test